blocks.md

```diff
@@ -164,7 +164,7 @@ Key | Values | Required | Default
 `warning` | Minimum usage, where state is set to warning. | No | `60`
 `critical` | Minimum usage, where state is set to critical. | No | `90`
 `interval` | Update interval, in seconds. | No | `1`
-`frequency` | Shows avg cpu frequency in GHz | No | `false`
```

Comment:
Don't remove this; instead, add that it is "deprecated in favour of the `format` key".